### DISCUSSION TITLE
sfdx force: package:version:promote - add validate only option

### DISCUSSION THREAD

**Discussion by arafesthain** [forcedotcom/cli#2383] [Ideas]
**Is your feature request related to a problem? Please describe.**

I'm always frustrated when I try to promote a version with my "master/main" branch pipeline and get "You already promoted a package version with this major.minor.patch version number. For a given major.minor.patch number, you can promote only one version."

**What are you trying to do**
promoting version of unlock package

**Describe the solution you'd like**
Add option validate only on the command line in order for the build pipeline to shift left (in the dev cycle) the error catching.

**Describe alternatives you've considered**
Automatically increment Z (patch) version

[a comment by an automated account is not shown]

**Comment by eneag-sf** [comment]
yes please! we need this.✅

[a comment by an automated account is not shown]
